config/puma.rb

```diff
@@ -9,7 +9,8 @@
 
 # Specifies the `environment` that Puma will run in.
 #
-environment ENV.fetch("RAILS_ENV") { "development" }
+env = ENV.fetch("RAILS_ENV") { "development" }
+environment env
```

Comment:
What's the benefit of this diff?

Reply:
I added it because we re-use the env variable later on